python/lsst/daf/butler/core/config.py

```diff
@@ -343,93 +258,30 @@ def fromYaml(cls, string: str) -> Config:
         c : `Config`
             Newly-constructed Config.
         """
-        return cls().__initFromYaml(string)
+        new_config = cls().__initFromYaml(string)
+        new_config._processExplicitIncludes()
+        return new_config
 
-    def __initFromFile(self, path: str) -> None:
+    def __initFromUri(self, path: str) -> None:
         """Load a file from a path or an URI.
 
         Parameters
         ----------
         path : `str`
-            Path or an URI to a persisted config file.
+            Path or a URI to a persisted config file.
         """
         uri = ButlerURI(path)
         if uri.path.endswith("yaml"):
```

Comment:
`uri.getExtension() == ".yaml"` instead?

Reply:
Yes. That would be better (I wrote that before getExtension existed).